examples/list/duckduckgoose/main.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/charmbracelet/lipgloss"
+	"github.com/charmbracelet/lipgloss/list"
+	"github.com/charmbracelet/lipgloss/tree"
+)
+
+func duckDuckGooseEnumerator(data tree.Data, i int, _ bool) (string, string) {
```

Comment:
I'm not a big fan of how a `tree.Data` is passed here for the enumerator. I believe this should simply be passed the `list` so that users don't confuse trees and lists.

Reply:
I think the enumerator API should simply be:

```go
func (l list.List, i int) string
```

Perhaps even (since you could theoretically query the list separately given the index, but not a huge fan of this personally):

```go
func (i int) string
```

Is there a possible way to achieve that?

Reply:
For the `last bool`, I believe this information could live on the `tree` and query-able by the index:

Something along the lines of (with a better name):
```
tree.IsLastChild(i)
```

Reply:
The `last bool` was indeed not needed anymore, removed it.

As for passing the list directly, I disagree for two reasons:

1. there is no `list.List` type, just a `list.New` that returns `tree.Tree`
2. the `tree.Tree` api is too large, IMO we should pass the smallest interface possible in order to do the job, which I think is is `Data` - or something even smaller